src/style.css

```
@@ -0,0 +1,126 @@
+.body {
+  margin: 0;
+  --color: #00acdc;
+  --star_size: 16px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card_img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
```

```suggestion
  margin-bottom: 40px;
```
32px can be padding for .card
you can just center horizontally image without horizontal margins